src/store/modules/compound.js

```diff
@@ -97,6 +102,7 @@ export default {
   actions,
   mutations,
   modules: {
+    substance,
```

Comment:
I thought this was the convention to use `dispatch` or `commit` from another module
```if (inc) dispatch("substance/loadForm", inc.shift()); ```
I would get this otherwise...
```[vuex] unknown local action type: substance/loadForm, global type: compound/definedcompound/substance/loadForm```

Reply:
I checked this out, you're right that it's trying to pull a local instance.  Replacing it with this seems to work.
```
if (inc) dispatch("substance/loadForm", inc.shift(), { root: true });
```

Reply:
looks right, I think the place that I had gotten twisted was making a `commit` from another module which requires a `null` arg in between to work like so
```
commit("substance/clearForm", null, { root: true });
```